docker/basic/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM pytorch/pytorch:1.5.1-cuda10.1-cudnn7-devel
+
+#install git
+RUN apt-get update && apt-get install -y --no-install-recommends \
+         git && \
+     rm -rf /var/lib/apt/lists/*
+
+#setup opencv
+RUN echo '#!/bin/bash\n\
+python -c "import cv2"\n\
+res=$?\n\
+if [ "$res" -eq "1" ]; then\n\
+    echo "Install libglib2.0 libsm6 libxext6 libxrender-dev for opencv"\n\
+    apt-get update\n\
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime\n\
+    apt-get install -y tzdata\n\
+    dpkg-reconfigure --frontend noninteractive tzdata\n\
+    apt-get -y install --no-install-recommends libglib2.0 libsm6 libxext6 libxrender-dev\n\
+fi\n'\
+>> setup_opencv.sh
+
+RUN sh setup_opencv.sh
+
+#setup apex
+RUN echo '#!/bin/bash\n\
+tmp_apex_path="/tmp/apex"\n\
+python -c "import apex"\n\
+res=$?\n\
+if [ "$res" -eq "1" ]; then\n\
```

Comment:
Same here. Just install Nvidia/apex. Maybe, `TORCH_CUDA_ARCH_LIST` can be and argument.